run_app.py

```diff
@@ -0,0 +1,83 @@
+import requests
```

Comment:
This script compares the performance between `app_temp_file.py` and `app_server.py`.